jspm.config.js

```diff
@@ -1,5 +1,8 @@
 SystemJS.config({
-  transpiler: "plugin-babel"
+  transpiler: "plugin-babel",
+  map: {
+    "feature.js": "github:viljamis/feature.js@1.0.1"
+  }
```

Comment:
I kind of liked the `import feature from 'viljamis/feature.js` path; it gives me some context around where the package is coming from.